index.bs

```diff
@@ -1605,6 +1606,34 @@ This algorithm will return either a non-negative 128-bit integer or an error.
 1. If the characters within |value| are not all [=ASCII hex digits=], return an error.
 1. Interpret |value| as a hexadecimal number and return as a non-negative 128-bit integer.
 
+<h3 dfn id="should-block-processing-for-reporting-origin-limit">Should processing be blocked by reporting-origin limit</h3>
```

Comment:
In a separate change, can we remove the `<h3>` associated with a bunch of these algorithms? Most of them don't need to show up in the document outline. I would just expect a single top-level heading like `Rate-limiting Algorithms` that covers the shared behavior. Individual ones don't need to given a header.

Reply:
Will move in a follow up PR.